src/web-ui/src/analytics/AnalyticsHandler.js

```diff
@@ -839,4 +856,108 @@ export const AnalyticsHandler = {
     googleAnalyticsEnabled() {
         return process.env.VUE_APP_GOOGLE_ANALYTICS_ID && process.env.VUE_APP_GOOGLE_ANALYTICS_ID != 'NONE';
     },
+
+    evidentlyEnabled() {
+        return process.env.VUE_APP_EVIDENTLY_PROJECT && process.env.VUE_APP_EVIDENTLY_PROJECT != 'NONE' &&
+                process.env.VUE_APP_EVIDENTLY_REGION && process.env.VUE_APP_EVIDENTLY_REGION != 'NONE' &&
+                process.env.VUE_APP_EVIDENTLY_ENDPOINT && process.env.VUE_APP_EVIDENTLY_ENDPOINT != 'NONE'
+    },
+
+    async getExternalExperiment(userId, feature) {
+        if (this.evidentlyEnabled()) {
+            try {
+                const cognitoUser = await Vue.prototype.$Amplify.Auth.currentAuthenticatedUser()
+                if (cognitoUser && cognitoUser.signInUserSession) {
+                    const creds = await Vue.prototype.$Amplify.Auth.currentUserCredentials();
+                    const evidentlyClient = new Evidently({
+                        region: process.env.VUE_APP_EVIDENTLY_REGION,
+                        endpoint: process.env.VUE_APP_EVIDENTLY_ENDPOINT,
+                        credentials: creds
+                    });
+
+                    const config = await evidentlyClient.evaluateFeature({
+                        entityId: userId,
+                        feature: feature,
+                        project: process.env.VUE_APP_EVIDENTLY_PROJECT
+                    }).promise();
+
+                    if (config.reason == 'EXPERIMENT_RULE_MATCH') {
+                        return {
+                            id: config.details.experiment,
+                            name: config.details.experiment,
+                            type: 'evidently',
+                            variationType: config.value.stringValue,
+                            variationIndex: (parseInt(config.variation.replace('Variation','')) - 1)
+                        }
+                    }
+                    else if (config.reason == 'DEFAULT') {
+                        console.log(`Evidently feature ${feature} configured but experiment rule was not matched; experiment likely not active/running`);
+                    }
+                }
+                else {
+                    console.log("User not signed so cannot call Evidently")
+                }
+            }
+            catch(error) {
+                // eslint-disable-next-line
+                console.error("Unable to evaluate feature", error)
+            }
+        }
+    },
+
+    async recordEvidentlyExposore(user, experiment) {
+        const dataCustom = `{
```

Comment:
Would it be more ergonomic to create an object and then call JSON.stringify for these events?